files/usr/bin/cinnamon-launcher

```diff
@@ -31,14 +31,27 @@ elif os.path.exists("/usr/bin/tint2"):
 
 
 def confirm_restart():
-    d = Gtk.MessageDialog(parent=None, flags=0, message_type=Gtk.MessageType.WARNING, buttons=Gtk.ButtonsType.YES_NO)
+    d = Gtk.Dialog(title=None, parent=None, flags=0)
+    d.add_buttons(Gtk.STOCK_NO, Gtk.ResponseType.NO,
+                  Gtk.STOCK_YES, Gtk.ResponseType.YES)
```

Comment:
It would be better not to use GtkStock in new code.

Reply:
Hi @JosephMcc 
thanks for your comment - Can you explain why using Gtk.STOCK_X is deprecated? I checked the library documentation and couldn't find anything related to it being deprecated.

OTOH, if not, would you prefer a simpl `_("Yes")` and `_("No")`?

Thanks

Reply:
Hello, everybody.

@norbusan: For an up-to-date documentation of Gnome libraries (Gtk, Clutter, GLib, Gio, etc.), exists [this on-line documentation](https://lazka.github.io/pgi-docs/) that can also be [downloaded for off-line use](https://github.com/lazka/pgi-docs) as a static HTML site, and also can be downloaded as [Devhelp books](https://github.com/pygobject/pgi-docs-devhelp). These documentations are extracted directly from the Debian testing/unstable source code of said libraries. Each documentation details when a function, constant, class, etc. has been introduced and/or deprecated.

Reply:
Thanks @Odyseus 
so I see now that the STOCK stuff is deprecated. The explanation for replacement is minimal, so would the following be what you have in mind?
```
    d.add_buttons("_No", Gtk.ResponseType.NO,
                  "_Yes", Gtk.ResponseType.YES)
```
?

Reply:
Oh, yes. Completely disregard my comments on `Gtk.MessageDialog`. I didn't see that you added more widgets to the dialog further down the code.

In regards to `gettext` on Python, it's pretty much the same as in any other language. You just use a call to `_()` with the strings that you want to be exposed for localization. The call to `gettext.install("cinnamon", "/usr/share/locale")` at the beginning of this file installs the function `_()` in Python’s built-ins namespace ([gettext module docs](https://docs.python.org/3/library/gettext.html#module-gettext)).

By the way, you also need to add the calls to `_()` to the strings at around line **43** of this file.

Reply:
Ok, thanks. I have changed to `_("Yes")` and `_("No")`, and added the same translation call to the other strings. Pushed to this PR.